sdk/cloudmachine/Azure.Projects/src/Azure.Projects.csproj

```diff
@@ -21,8 +22,13 @@
     <PackageReference Include="Azure.AI.Projects" />
     <PackageReference Include="Azure.AI.OpenAI" />
     <PackageReference Include="Azure.Search.Documents" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions"/>
-    <PackageReference Include="System.Memory.Data" VersionOverride="8.0.0"/>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" />
+    <PackageReference Include="System.ClientModel" />
+    <PackageReference Include="System.Memory.Data" VersionOverride="8.0.0" />
```

Comment:
We should try to remove this version override if possible.

Reply:
I need a feature that is only in 8.0. Would you want me to update the whole repo?

Reply:
Yes in general we try to keep the repo aligned.

Reply:
I will try to do it in a subsequent PR. I am having problems merging this one and I don't want to make it 10x bigger.